gazebo/dashing-gazebo9/post_rosdep_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Only set up Gazebo's repository
+set -e
```

Comment:
could you add `+x` please?

Reply:
You mean `-x`? It's not set for Kinetic's `post_rosdep_install.sh`.

Reply:
-x makes the script print everything it's doing to the console, it's useful for debugging if this script sometimes has issues.